lib/server.js

```diff
@@ -12,6 +12,7 @@ const startServer = (config, callback) => {
   if (callback == null) callback = Function.prototype;
   const serverOpts = config.server || {};
   const port = parseInt(serverOpts.port, 10);
+  const host = config.server.hostname || 'localhost';
```

Comment:
What I mean is, can this be getting from `brunch s -h whatever` and not app config? There probably could be a way to access that from auto-reload-brunch... A thing like this, doesn't feel like it belongs into app config